luaui/Widgets/gui_gridmenu.lua

```diff
@@ -2114,13 +2122,7 @@ function widget:MousePress(x, y, button)
 							if isPregame then
 								setPreGamestartDefID(cellcmds[cellRectID].id * -1)
 							elseif spGetCmdDescIndex(cellcmds[cellRectID].id) then
-								Spring.SetActiveCommand(
```

Comment:
And here, for reference: `Spring.SetActiveCommand(cmd, 1, false, true,Spring.GetModKeyState())`